cal-itp/benefits
# Refactor: reCAPTCHA copy + redesign

closes #2537

## What this PR does
### Actual feature
- Adds reCAPTCHA text to the 2 form pages on Benefits user-facing flow
- Removes reCAPTCHA icon from the 2 form pages

### Design changes
- Removes icons from the main flow. Icons remain on Error pages.
- Makes the main flow plus error pages (minus Enrollment Success) all 6 columns.
- Makes all the headers left-aligned, across desktop and mobile. The design is now much more "mobile-first" now, with zero alignment/column-width or font size changes between mobile and desktop.
- Buttons are 6-col wide. 
- Get started with Login.gov button is now the same height as the other CTAs.
- Agency Card form: The form helper text now has correct letter-spacing.
- Updates the external link icon CSS to allow for the icon to change sizes when font size changes

### Under the hood refactors
- Removes `media-item` CSS and templates. Now there's an `eligibility-item` that is only used for that purpose.
- Refactors `base.html` to be simpler, and have fewer things to override. Fewer empty divs delivered to the code (should be none). Details: https://github.com/cal-itp/benefits/pull/2588
- Error page refactors: https://github.com/cal-itp/benefits/pull/2585
- Naming conventions around `base` templates: All templates inherit from `base`, which may inherit from another intermediary base (like `error-base`). If a template is inheriting from `base` and meant for other pages to inherit, put the term `base` in the filename.


## How to test locally
- Use https://github.com/cal-itp/benefits/pull/1071 instructions to test in Dev. The reCAPTCHA test environment key/secret are in LastPass, under Benefits reCAPTCHA configuration.
- Make sure to test CalFresh flow
- No regressions on Admin In Person flow

## How to test on `dev`
@angela-tran is adding the reCAPTCHA env vars to dev, so we will be able to test on dev.
